app/modules/session/session.server.ts

```diff
@@ -75,4 +75,5 @@ export async function requireUserSession(request: Request) {
   if (!session) {
     return redirect('/login');
   }
+  return session;
```

Comment:
this one was also implemented on the merge to branch (not main), but I don't wanna do a rebase, so just keeping it here.